packages/nodebox/package.json

```diff
@@ -3,7 +3,15 @@
   "version": "0.0.39",
   "description": "Public API of Nodebox",
   "main": "./build/index.js",
+  "module": "./build/index.mjs",
   "typings": "./build/index.d.ts",
+  "exports": {
+    "/": {
```

Comment:
Shouldn't this be `.`? Pretty sure `/` isn't a valid export, at least I hope it isn't otherwise I need to change our resolver again 😅

Reply:
oh yeah, it's always confusing if it's `.` or `/` at the begining 😁 
https://github.com/preactjs/preact/blob/master/package.json#L13

Reply:
Yeah, the `exports` if the most confusing one. `default`, `imports`, `node`, `browser`. They throw all the new stuff into it 😂